docs/wiki/Cryptography/bls.md

```diff
@@ -0,0 +1,232 @@
+<!-- @format -->
+
+# Boneh-Lynn-Shacham (BLS) signature
+
+### TLDR;
+
+- Proof of stake protocols use digital signatures to identify their participants and hold them accountable.
```

Comment:
The term should proof-of-stake 
https://ethereum.org/en/contributing/style-guide/content-standardization/#proof-of

```suggestion
- Proof-of-stake protocols use digital signatures to identify their participants and hold them accountable.
```

Reply:
Also, maybe I would rephrase this or add another point more specific to Eth CL. Something along 'Validators in Beacon Chain work with BLS to participate in the consensus, post attestations, initiate exit,..'